### PR TITLE
PLAT-5748 Add initial GH config to enable the workflow

### PR DESCRIPTION
**Pull request by shashanksinha89**
## Ticket

PLAT-5748

## Type of PR

- [ ] Bugfix
- [x] New feature
- [ ] Minor changes

## Changes introduced

_Add initial GH config to enable the workflow_

### CODE REVIEW

File: .github/workflows/release.yml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  push:
+    branches:
+      - 'master'
+      - 'develop'
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```
Comment (mirr254) [inline .github/workflows/release.yml:13]: I don't understand why only `checkout` is the only step here. Nothing happens after checking out the code?

Comment (shashanksinha89) [reply .github/workflows/release.yml:13]: It's just to enable the GH workflow. It will change with final PR.

Comment (mirr254) [reply .github/workflows/release.yml:13]: Ok. Why not include all the work in one PR and then review all of it? Since it's one ticket?

**Comment by mirr254** [comment]
@shashanksinha89 I am a bit confused here. The PR description is this https://crowdbotics.atlassian.net/browse/PLAT-9071 which is the model-builder, but this ticket is for https://crowdbotics.atlassian.net/browse/PLAT-5748 module releases. 

Not sure if they are related.

File: .github/workflows/release.yml (hunk 1 above)
Comment (shashanksinha89) [reply .github/workflows/release.yml:13]: As I need to test workflow events with correct branches and test it with dispatch event while testing the workflow. Merging it to default branch makes it easy to clone it PR branches. Also this PR won't make any impact to existing environments.

**Comment by shashanksinha89** [comment]
> @shashanksinha89 I am a bit confused here. The PR description is this https://crowdbotics.atlassian.net/browse/PLAT-9071 which is the model-builder, but this PR is for https://crowdbotics.atlassian.net/browse/PLAT-5748 module releases.
> 
> Not sure if they are related.

don't worry, I have updated the ref ticket.

**Review by mirr254 — APPROVED** [review]

**Pull request merged**